api/src/pcapi/core/search/commands/indexation.py

```diff
@@ -226,3 +229,23 @@ def _reindex_all_venues(algolia_batch_size: int, max_venues: int) -> None:
         venue_ids = [venue.id for venue in venue_chunk]
         search.reindex_venue_ids(venue_ids)
         logger.info("Reindex %d eligible venues from page %d", len(venue_ids), page)
+
+
+@blueprint.cli.command("reindex_offers_if_ean_booked_recently")
+@click.option("--since", help="time (in hours) since last booking", type=int, default=24)
+@click.option("--batch-size", help="Batch size (Algolia)", type=int, default=10_000)
+@click.option("--max-offers", help="Max number of offers (total)", type=int, default=10_000)
+def reindex_offers_if_ean_booked_recently(since: int, batch_size: int, max_offers: int) -> None:
+    """
+    Reindex offers if they have been booked recently.
+    Currently, we consider that an offer has been booked recently if it has been booked at least once in the last 30 days.
```

Comment:
[question / doc] "if it has been booked at least once in the last 30 days"... mais la valeur par défaut de `since` est de 24 heures. Je me suis un peu perdu, mais si j'ai bien compris : cette commande est lancée via cron chaque jour, donc à chaque fois on veut réindexer les offres réservées depuis la veille. Ça c'est l'argument `since` (24 heures). Et, lorsque l'on réindexe chaque offre, on envoie à Algolia le nombre total de réservations (créées depuis 30 jours) de l'EAN correspondant.

Donc je trouve la 2ème phrase de la docstring ambigüe. Proposition : 

```suggestion
    Reindex offers that have been booked recently (24 hours by default, because this
    command is run by a cron every day).

    This command is needed to reindex offers that have the same EAN than every offer
    that has been booked in the last 24 hours, so that each of them have an up-to-date
    total number of bookings (per EAN) in the last 30 days (see `last30DaysBookings`
    in Algolia serialization).
```

Qu'en penses-tu ?